src/en/space-station-14/departments/silicon.md

```diff
@@ -0,0 +1,102 @@
+# Silicons
+A collection of artificial entities that follow programmed laws to assist their creators.
+
+## Concept
+### What is a Silicon?
+Silicons are a neutral faction of robots and artificial intelligences commonly found aboard Nanotrasen stations. These artificial entities are typically constructed to fulfil a specific purpose and generally excel in this capacity. For example, a Silicon built to support the engineering department can easily repair hull breaches and damaged infrastructure; a Silicon built to assist the medical department can diagnose different humanoid ailments and treat them; and a Silicon constructed for janitorial duties excels at cleaning floors. However, the specialized construction of Silicons makes performing tasks outside of their design specifications exceptionally difficult. As such, they are not intended to be a replacement for, or an upgrade to, humanoid crew members.
+
+Silicons are typically bound by programmed laws (see below), which place constraints and certain expectations on how Silicon players should behave. These laws are not expected to be mechanically enforced; consequently, all Silicon roles intrinsically require some degree of role-playing by those who play them.
+
+At present, there are three archetypes of Silicon: simple robots (bots), cyborgs (borgs), and artificial intelligences (station and personal AIs). Robots typically have very limited capabilities, and it is not expected for players to take direct control of them outside of ghost roles.
```

Comment:
This feels like a bit of a turtle/tortoise situation. Not all robots should be silicons, however all silicons ARE robots.

We should add an addendum to clarify this idea, more so if we decide later that certain robots should indeed become full silicons or downgrade a silicon into a robot.